app/lib/bk/compat/parsers/harness.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 require_relative '../translator'
-require_relative '../pipeline'
-require_relative '../pipeline/step'
+require_relative '../models/pipeline'
+require_relative '../models/steps/group'
```

Comment:
This file also uses the command step class so it should include the dependencies explicitly:

```suggestion
require_relative '../models/pipeline'
require_relative '../models/steps/command'
require_relative '../models/steps/group'
```

Reply:
Added 👍